src/AstroImages.jl

```diff
@@ -103,12 +102,25 @@ function AstroImage(file::String)
     if ext > 1
        	@info "Image was loaded from HDU $ext"
     elseif ext == 0
-        error("There are no ImageHDU extensions in \"$file\"")
+        error("There are no ImageHDU extensions in '$(fits.filename)'")
     end
-    out = AstroImage(Gray, fits, ext)
-    close(fits)
-    return out
+    return ext
+end
+function AstroImage(files::NTuple{N,String}) where {N}
+    data = Array{Array{Float64,2}}(undef,N)
+    wcs = Array{WCSTransform}(undef,N)
+    for (idx,file) in enumerate(files)
+        fits = FITS(file)
+        ext = indexer(fits)
+        data[idx] = float.(read(fits[ext]))
```

Comment:
Also. here I'm storing the image data as `float` as it is going to be converted to float otherwise while rendering and this will prevent type instability in the `NTuple` `data` filed.